jax/numpy/lax_numpy.py

```diff
@@ -3089,6 +3089,18 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
   return true_divide(dot(X, X_T.conj()), f).squeeze()
 
 
+@_wraps(onp.convolve)
+def convolve(x, y, mode='full'):
+  x_jax = lax.reshape(x,(1,1,len(x)))
```

Comment:
Minor style nit: elsewhere in JAX, the style is to put a space after most commas, so here:
` x_jax = lax.reshape(x, (1, 1, len(x)))`
Same everywhere.